src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmTestResultAction.java

```diff
@@ -35,16 +37,22 @@
 
     private static final int DefaultUpdateInterval = 30 * 1000;
 
-    private PrintStream log;
     private AWSDeviceFarmTestResult result;
 
-    public AWSDeviceFarmTestResultAction(AbstractBuild<?, ?> owner, AWSDeviceFarmTestResult result, PrintStream log) {
-
-        super(owner);
-        this.log = log;
+    public AWSDeviceFarmTestResultAction(hudson.model.Run<?, ?> owner, AWSDeviceFarmTestResult result) {
```

Comment:
It can be refactored later to get rid of `hudson.model.Run<?, ?> owner` in API at all